src/shared/components/palette/case-link/components/before-you-start/before-you-start.component.ts

```diff
@@ -35,4 +34,10 @@ export class BeforeYouStartComponent {
       navigateToNextPage: true
     });
   }
+
+  public onBack(): void {
+    this.router.navigate(['cases', 'case-details', this.linkedCasesService.caseId]).then(() => {
+      window.location.hash = 'Linked cases';
```

Comment:
1. suppose it shouldn't use blank string in the hash? 
2. and does it have that hash key in the html template?

Reply:
As far as I know, there is no hash key for the linked  cases and we usually redirect to the landing page tab.

Reply:
so does that mean this window.location.hash never work can we remove it?